scripts/west_commands/ncs_west_helpers.py

```diff
@@ -13,9 +13,15 @@
 from pathlib import PurePath
 import re
 import textwrap
-
-import editdistance
-import pygit2
+import sys
+
+try:
```

Comment:
In case anyone is concerned, this doesn't break normal operation, because west won't import extension command modules until the user actually runs them.